starlark/library.go

```diff
@@ -888,6 +889,32 @@ func print(thread *Thread, b *Builtin, args Tuple, kwargs []Tuple) (Value, error
 	return None, nil
 }
 
+// TODO: add link to spec.md
+func fail(thread *Thread, b *Builtin, args Tuple, kwargs []Tuple) (Value, error) {
+	if len(kwargs) > 0 {
+		return nil, fmt.Errorf("fail does not accept keyword arguments")
```

Comment:
If you use UnpackPositionalArgs, you don't need this check.

But given the behavior of fail I proposed, you don't need the Unpack functions. Just grab starlark.AsString(args[0]), and all of kwargs.